junebug/channel.py

```diff
@@ -144,13 +146,31 @@ def get_all(cls, redis):
 
     @classmethod
     @inlineCallbacks
-    def send_message(cls, id, message_sender, msg):
-        '''Sends a message. Takes a junebug.amqp.MessageSender instance to
+    def send_message(cls, id, sender, msg):
+        '''Sends a message. Takes a :class:`junebug.amqp.MessageSender` instance to
         send a message.'''
-        message = TransportUserMessage.send(
-            **cls.message_from_api(id, msg))
-        queue = '%s.outbound' % id
-        msg = yield message_sender.send_message(message, routing_key=queue)
+        msg = cls.message_from_api(id, msg)
+        msg.setdefault('to_addr', None)
```

Comment:
Is this required? In `message_from_api` we set `to_addr` to `msg.get('to')` which will return `None` if `to` doesn't exist.